actfw_core/capture.py

```diff
@@ -85,10 +86,10 @@ def cmp(config):
         config = None
         fmts = [expected_format] + fallback_formats
         for fmt in fmts:
-            candidates = self.video.lookup_config(width, height, framerate, fmt, expected_format)
+            candidates = self.video.lookup_config(width, height, 1 if format_selector == V4LCameraCapture.FormatSelector.MAXIMUM else framerate, fmt, expected_format)
```

Comment:
Seems better to re-calculate framerate at near the top of constructor and use it as-is here:

```python
        super(V4LCameraCapture, self).__init__()
        if format_selector == V4LCameraCapture.FormatSelector.MAXIMUM:
            framerate = 1  # ignored in MAXIMUM mode
```

Reply:
already resolved: https://github.com/Idein/actfw-core/pull/4/commits/c8e94846c1902c9bee9c899130b57b19560a61b2